### PR TITLE
Add tag output~

### PR DESCRIPTION
**Pull request by monkeyWie**
Hello,as you said,parsing the `refs/tags/xxx` is a matter in workflow,so I re open this PR.

### CODE REVIEW

**Comment by csexton** [comment]
Oh, this is handy 😍

File: dist/index.js [hunk 1]
```diff
@@ -7844,53 +7844,54 @@ exports.request = request;
...
+    // Set the output variables for use by other actions: https://github.com/actions/toolkit/tree/master/packages/core#inputsoutputs
+    core.setOutput('id', releaseId);
+    core.setOutput('html_url', htmlUrl);
+    core.setOutput('upload_url', uploadUrl);
+    core.setOutput('tag', tag);
+  } catch (error) {
+    core.setFailed(error.message);
+  }
+}
+
```
Comment (jnovack) [inline dist/index.js]: What happened here? Spaces to tabs or vice-versa?

Comment (deancn) [reply dist/index.js]: `dist/index.js`  Is this file auto-generated by build?   @monkeyWie  Can you help fix it?

Comment (monkeyWie) [reply dist/index.js]: Should  I revert`dist / index.js` and build it by author self?

Comment (deancn) [reply dist/index.js]: @IAmHughes  help merge this pr, pls.

Comment (deancn) [reply dist/index.js]: @IAmHughes   Help to merge this pr, pls.

**Comment by monkeyWie** [comment]
@jnovack @IAmHughes I have resolved the conflict，please review again.

File: dist/index.js (hunk 1 above)
Comment (fleskesvor) [reply dist/index.js]: @deancn `dist/index.js` is created and added to the commit by `npm run precommit`, which is run by a pre-commit Git hook added by husky when you `npm install` the project. You should not edit it yourself, but instead fix errors in `src/`.

**Comment by pplavetzki** [comment]
Any update on this?  Would be most useful to only have the tag_name and not the /ref/tags/tag_name

**Comment by balcsida** [comment]
Hi folks,

Could you please check on this PR, please?
This would greatly improve my workflows - uploading versioned ZIP files is sick :D

Kind regards,
@balcsida

**Comment by jnovack** [comment]
Won't review until all conflicts are resolved.  `dist/index.js` looks like a mess in the diff editor.

**Comment by monkeyWie** [comment]
Reopen a PR:https://github.com/actions/create-release/pull/97

**Pull request closed without merging**